packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -446,36 +446,13 @@ class OAuthController {
     }
 
     public async oauthCallback(req: Request, res: Response, _: NextFunction) {
-        const account = (await getUserAndAccountFromSession(req)).account;
         const { state } = req.query;
 
-        const log = {
-            level: 'info' as LogLevel,
-            success: false,
-            action: 'oauth' as LogAction,
-            start: Date.now(),
-            end: Date.now(),
-            timestamp: Date.now(),
-            connection_id: '',
-            provider_config_key: '',
-            account_id: account.id
-        };
-
         if (state == null) {
             const errorMessage = 'No state found in callback';
             const e = new Error(errorMessage);
 
-            const errorActivityLog = await createActivityLog(log);
-
-            await createActivityLogMessage({
-                level: 'error',
-                activity_log_id: errorActivityLog as number,
-                content: WSErrBuilder.MissingConnectionId().message,
-                timestamp: Date.now(),
-                params: {
-                    ...errorManager.getExpressRequestContext(req)
-                }
-            });
+            logger.log('error', errorMessage);
 
             errorManager.report(e, { metadata: errorManager.getExpressRequestContext(req) });
```

Comment:
I think `errorManager.report` already logs the error, so I don't think we need the previous `logger.log` call.